source/change-streams/tests/unified/change-streams-errors.yml

```diff
@@ -0,0 +1,159 @@
+description: "change-streams errors"
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents: [ commandStartedEvent ]
+      ignoreCommandMonitoringEvents: [ killCursors ]
+      useMultipleMongoses: false
+  - client:
+      id: &globalClient globalClient
+      useMultipleMongoses: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+  - database:
+      id: &database1 database1
+      client: *client0
+      databaseName: *database1
+  - collection:
+      id: &collection1 collection1
+      database: *database1
+      collectionName: *collection1
+  - database:
+      id: &globalDatabase0 globalDatabase0
+      client: *globalClient
+      databaseName: *database0
+  - collection:
+      id: &globalCollection0 globalCollection0
+      database: *globalDatabase0
+      collectionName: *collection0
+  - database:
+      id: &globalDatabase1 globalDatabase1
+      client: *globalClient
+      databaseName: *database1
+  - collection:
+      id: &globalCollection1 globalCollection1
+      database: *globalDatabase1
+      collectionName: *collection1
+  # Some tests run operations against db1.coll0 or db0.coll1
+  - collection:
+      id: &globalDb1Collection0 globalDb1Collection0
+      database: *globalDatabase1
+      collectionName: *collection0
+  - collection:
+      id: &globalDb0Collection1 globalDb0Collection1
+      database: *globalDatabase0
+      collectionName: *collection1
+
+initialData:
+  - collectionName: *collection0
+    databaseName: *database0
+    documents: []
+tests:
+  - description: "The watch helper must not throw a custom exception when executed against a single server topology, but instead depend on a server error"
+    runOnRequirements:
+      - minServerVersion: 3.6.0
+        topologies:
+          - single
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          options: {}
```

Comment:
So I think `createChangeStream` options such as `batchSize` can appear under `arguments` adjacent to things like `pipeline`. Like:
```
operations:
  - name: createChangeStream
     object: *collection0
     arguments:
      pipeline: []
      batchSize: 1
```
The unified test format [describes](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#id4) this helper as having all the same options as the corresponding `watch` method. Having the options embedded under `options:` seems odd to me (we can avoid having `options: {}` whenever there are no options, too).

Reply:
Done.